documentation/README.md

```diff
@@ -0,0 +1,72 @@
+# Documentation
+
+The documentation is built using [Docusaurus 2](https://docusaurus.io/). The deployment is done through a centralized build from [IOTA WIKI](https://github.com/iota-community/iota-wiki). To run a local instance the [IOTA WIKI CLI](https://github.com/iota-community/iota-wiki-cli) is used.
+
+## Prerequisites
+
+- [Node.js v14.14+](https://nodejs.org/en/)
+- [yarn](https://yarnpkg.com/getting-started/install)
+
+## Installation
+
+```console
+yarn
+```
+
+This command installs all necessary dependencies.
+
+## Local Development
+
+```console
+yarn start
+```
+
+This command starts a local, wiki themed development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+## Including .md file
+
+```console
+{@import <file path>}
+```
+
+Example:
+
+```console
+{@import ../../../../bindings/wasm/docs/api-reference.md}
+```
```

Comment:
I must say I agree, this file is copy-pasted from another project.  I thought perhaps you and Kevin didn't want to complicate users with some React.  Though to be honest it's pretty much the same except the imports are scattered all over the place with one method, which usually is hard to maintain.

Reply:
I actually also just used it everywhere because identity had it in there and other projects used it.
My plan was to eventually remove it anyway and add all this stuff to the wiki as a guide. So we could link our devs there if they want to see what is possible and what we allow.
Same for default plugins which are available for all repos.

Also not using react came mainly from the editor. But I guess we just drop that project :trollface: 
But tbh we always agreed that we should allow more freedom in the external repo docs